global/README.md

```diff
@@ -8,61 +8,66 @@
 
 ## General Rules
 
-* The use of external tools that modify the game in any way, shape, or form are
+* The use of external tools that modify the game in any way, shape, or form is
 forbidden.
-* The as of now Android-exclusive 1.2.6.* duping glitch is banned from all
-categories unless explicitely stated otherwise.
+* The, as of now, Android-exclusive 1.2.6.* duping glitch is banned from all
```

Comment:
unrelated to the changes but this should be Mobile-exclusive (iOS can change versions, and has 1.2.6 available (https://70.glitch.me/)
```suggestion
* The, as of now, Mobile-exclusive 1.2.6.* duping glitch is banned from all
```